src/resmom/mom_main.c

```diff
@@ -5008,6 +5006,17 @@ read_config(char *file)
 		 */
 		cpu_raresync();
 #endif	/* MOM_CPUSET */
+
+		if (joinjob_alarm_time == -1)
+			update_joinjob_alarm_time = 1;
+		else
+			update_joinjob_alarm_time = 0;
+
+		if (job_launch_delay == -1)
+			update_job_launch_delay = 1;
```

Comment:
Do you want to use 1 here, or DEFAULT_JOB_LAUNCH_DELAY?

Reply:
This is for setting the boolean 'update_job_launch_delay' to 1 and not the 'job_launch_delay' timeout value, to tell pbs_mom to update the job_launch_delay value when execjob_prologue hook alarm value changes. This is because at this point, there was no explicit $job_launch_delay value given in mom's config file since job_launch_delay is still set to -1 after processing the file inside read_config().

Reply:
Thanks for clarifying.